docusaurus/docs/get-started/get-started.mdx

```diff
@@ -135,82 +167,52 @@ Depending on the answers you gave to the prompts, there should now be a structur
-
-### Do you want a backend part of your plugin?
-
-App and data source plugins can have a backend component written in Go. Backend plugins offer powerful features such as:
+Builds backend plugin binaries for Linux, Windows and Darwin.
 
-- Enable [Grafana Alerting](https://grafana.com/docs/grafana/latest/alerting/) for data sources.
-- Connect to non-HTTP services to which a browser normally can’t connect. For example, SQL database servers.
-- Keep state between users. For example, query caching for data sources.
-- Use custom authentication methods and/or authorization checks that aren’t supported in Grafana.
-- Use a custom data source request proxy. To learn more, refer to [Backend developer resources](/introduction/backend-plugins#resources).
+### `run build`
 
-### Do you want to add Github CI and Release workflows?
+Compiles and bundles the project using Webpack in production mode.
 
-Add [Github workflows](/create-a-plugin/develop-a-plugin/set-up-github-workflows) to your development cycle to help catch issues early or release your plugin to the community.
+### `run dev`
 
-### Do you want to add a Github workflow to automatically check Grafana API compatibility on PRs?
+Runs Webpack in watch mode for development, continually monitoring for changes.
 
-Add a [Github workflow](/create-a-plugin/develop-a-plugin/set-up-github-workflows#the-compatibility-check-is-compatibleyml) to regularly check that your plugin is compatible with the latest version of Grafana.
+### `run typecheck`
 
-## Key CLI commands
+Performs a type-checking process on the frontend code using TypeScript.
 
-After `create-plugin` has finished, you can run built-in commands in the shell:
+### `run server`
 
-### <SyncCommand cmd="run dev" />
+Launches the https://grafana.com/developers/plugin-tools/get-started/set-up-development-environment using Docker.
 
-Builds the plugin in _development mode_ and runs in _watch mode_. Rebuilds the plugin whenever you make changes to the code. You'll see build errors and lint warnings in the console.
+### `run sign`
 
-### <SyncCommand cmd="run test" />
+Signs the Grafana plugin using the latest version of `@grafana/sign-plugin`.
 
-Runs the tests and watches for changes.
+:::info
 
-### <SyncCommand cmd="run build" />
+For a complete list of available CLI commands, refer to the [CLI reference](/get-started/prompt-cli-reference#cli-commands).
 
-Creates a production build of the plugin that optimizes for the best performance. Minifies the build and includes hashes in the filenames.
+:::
 
-<h3>
-  <code>mage -v</code>
-</h3>
+## Next steps
 
-Builds backend plugin binaries for Linux, Windows and Darwin.
+- Start your plugin journey with one of our [plugin development tutorials](/tutorials/).
+- Learn how to [develop and extend](/create-a-plugin/) its functionality.
+- Review the [Examples](https://github.com/grafana/grafana-plugin-examples), [sign](/publish-a-plugin/sign-a-plugin) to learn about good practices.
```

Comment:
```suggestion
- Review the [Examples](https://github.com/grafana/grafana-plugin-examples), to learn about good practices.
- Learn how to package(link), sign(link) and publish(link) your plugin to the Grafana plugin catalog(link). 
```
In the last bullet, sign feels out of place, also not sure why Examples is capitalised?

Reply:
good suggestions. done. Examples shouldn't have been capitalized; we capitalize the titles of help topics but in this case it wasn't a help topic.